demo/kubernetes/README.md

```diff
@@ -43,12 +43,12 @@ This generates a discovery token so the nodes can find each other and then start
 
 First let's setup access to the rook cluster via the command line client.  You can download a pre-built from [github releases](https://github.com/rook/rook/releases) or [build from source](https://github.com/rook/rook/blob/master/build/README.md). (If you are not using coreos-kubernetes vagrant substitute the ip from above that you put in the `rook.yml` for `ROOK_API_SERVER_ENDPOINT`)
 
-    export ROOK_API_SERVER_ENDPOINT=172.17.4.101:8124
+    export ROOK_API_SERVER_ENDPOINT=172.17.4.201:8124
```

Comment:
no, this is just hitting the worker node instead of the controller node. Since increasing the memory in the controller node this shouldn't actually make a difference